src/core/CoreModuleDirector.ts

```diff
@@ -99,33 +115,32 @@ export class CoreModuleDirector {
 
     const modelStores = this.getModelStores();
 
-    const getModelName = (subscription: SupportedSubscription) => {
-      if (subscription.type === 'Email') {
-        return ModelName.EmailSubscriptions;
-      } else if (subscription.type === 'SMS') {
-        return ModelName.SmsSubscriptions;
-      } else {
-        return ModelName.PushSubscriptions;
-      }
-    };
-
     subscriptions.forEach(async (subscription) => {
-      const modelName = getModelName(subscription);
+      const modelName = ModelName.Subscriptions;
```

Comment:
Since this is always `ModelName.Subscriptions` we don't need this `const` here. I see it being used in only two spots below, would could probably just refer to `ModelName.Subscriptions` directly instead of using a `const`, or at least move it down there.